src/bitgo.js

```diff
@@ -206,6 +206,7 @@ var BitGo = function(params) {
         // browser (browserify sets process.browser).
         req.set('User-Agent', self._userAgent);
       }
+      req.timeout(process.env.BITGO_TIMEOUT || 60000);
```

Comment:
Since the server timeout is 5 minutes, I would prefer setting the default timeout to 5 minutes plus a little bit.